app/views/site/_search_options.html.erb

```diff
@@ -49,6 +49,9 @@
             </div>
             <%= link_to "<span class='fas fa-chevron-left'></span> Back".html_safe, "javascript:", class: "btn pull-right #{action_name == 'study' ? 'btn-default disabled' : 'btn-warning'}", id: 'clear-gene-search', title: 'Clear gene search and return to cluster scatter plots', data: {toggle: 'tooltip'} %>
             <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
+
+                intializeAutocomplete($('#search_genes'), <%= raw @unique_genes %>);
```

Comment:
```suggestion
                initializeAutocomplete('#search_genes', <%= raw @unique_genes %>);
```

Using a selector would be less tightly coupled with jQuery, and a tad shorter.

It also ensures what we're consistent with the function signature (which says `selector`, not something like `jqObject`).

Reply:
Agreed!